gateway/Dockerfile

```diff
@@ -74,12 +74,13 @@ ENV VITE_OSRD_GIT_DESCRIBE=${OSRD_GIT_DESCRIBE}
 COPY --from=front_src package.json package-lock.json /app/
 
 # If SKIP_FRONT is set to 1, skip the front build
-RUN if [ "$SKIP_FRONT" = "1" ]; then exit 0; else npm ci; fi
+RUN --mount=type=cache,target=/root/.npm \
+    if [ "$SKIP_FRONT" = "1" ]; then exit 0; else npm ci; fi
 
 # Generate the licenses file and build
 COPY --from=front_src . /app
 ENV NODE_OPTIONS="--max_old_space_size=4096"
-RUN if [ "$SKIP_FRONT" = "1" ]; then \ 
+RUN if [ "$SKIP_FRONT" = "1" ]; then \
```

Comment:
Doesn't this need the cache too?

Reply:
Sure, but doesn't building need the downloaded packages

Reply:
Yes, but the installed packages are in `node_modules`. `~/.npm` contains a cache.

Reply:
Alright then :)